restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/DialRecordingTest.java

```diff
@@ -276,6 +278,93 @@ public synchronized void testDialClientAlice_BobDisconnects() throws Interrupted
         assertEquals(0, liveCallsArraySize);
     }
 
+    @Test
+    @Category(FeatureAltTests.class)
+    public synchronized void testDialClientAlice_BobDisconnectsAndRemoveRecording() throws InterruptedException, ParseException {
```

Comment:
i think we are adding too much testing for this feature, not sure what these tests are adding . I see maybe one of these tests mixing actual call with recording API usage, but 3 of them seems to much. I would prefer to invest more on unit testing,rather than integration testing

Reply:
See my previous answer. I think 3 test cases is the minimum